docs/dev-tools/painlesslab/index.asciidoc

```diff
@@ -0,0 +1,22 @@
+[role="xpack"]
+[[painlesslab]]
+== Painless Lab
+
+beta::[]
+
+The Painless Lab is an interactive code editor that lets you test and
+debug Painless scripts in real-time. You can use the Painless scripting
+language to create {kib} scripted fields,
```

Comment:
Can we insert the links into these text instead of listing the bullets at the bottom?

You can use the Painless scripting language to create [Kibana scripted fields](), process [reindexed data](), define complex [Watcher conditions](), and work with data in other contexts.